LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ConcentratorDeduplication.cs

```diff
@@ -88,15 +88,7 @@ private bool EnsureFirstMessageInCache(string key, LoRaRequest loRaRequest, out
             return false;
         }
 
-        internal static string CreateCacheKey(LoRaRequest loRaRequest)
-            => loRaRequest.Payload switch
-            {
-                LoRaPayloadData asDataPayload => CreateCacheKey(asDataPayload),
-                LoRaPayloadJoinRequest asJoinPayload => CreateCacheKey(asJoinPayload),
-                _ => throw new ArgumentException($"Provided request is of type {loRaRequest.GetType()} which is not valid for deduplication.")
-            };
-
-        private static string CreateCacheKey(LoRaPayloadData payload)
+        internal string CreateCacheKey(LoRaPayloadData payload)
```

Comment:
This can be static.

Reply:
Changed, e987b029bcbb7d57399eac92758876fe11d6a2f1